release.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+echo "WARNING: This will reset your working directory!"
+echo "Please make sure all your changes are committed and merged into master."
+read -p "Are you sure you want to continue? " -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+    echo "Preparing to build..."
+    {
+        rm -r build & rm -r dist
+        git reset --hard
+        git checkout master
+        git pull --ff
+    } &> /dev/null
+    echo "Building new release..."
+    {
+        python setup.py bdist_wheel
+        WHEEL="$(ls -t1 dist/ | head -n 1)"
+    } &> /dev/null
+    echo "New release created:" $WHEEL
+    read -p "Would you like to upload to PyPi? " -n 1 -r
+    echo
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        {
+            twine upload dist/$WHEEL
+        } &> /dev/null
+        echo "Upload complete."
+    fi
+    {
+        rm -r build & rm -r dist
+    } &> /dev/null
+fi
```

Comment:
Generally, end files with newline.

Reply:
I'm guessing this is the resolution to the PyPi issue we were experiencing before?

Reply:
This was what was used to generate the artifacts and upload to PyPi

Reply:
An update to the previous version deployment to PyPi. The issue was as we suspected, a py3 artifact was not generated resulting in only downloading 3.8.
Once this is up hopefully my system will generate a 3.10 version